homeassistant/components/light/hue.py

```diff
@@ -161,12 +171,26 @@ def name(self):
     @property
     def brightness(self):
         """ Brightness of this light between 0..255. """
-        return self.info['state']['bri']
+        if 'bri' in self.capabilities:
+            return self.info['state']['bri']
```

Comment:
Good point, didn't test that.
If it turns out to work, i'll remove the capabilities portion.
If it doesn't ill improve the code of capabilities, it is a bit gnarly